admin-policy.md

```diff
@@ -0,0 +1,83 @@
+
+## 6. Removal of Admins
+
+1. Administrators may be removed by an unanimous vote among the other current admins. The current administrators may choose to hold a temporary election or wait until the next election if an administrator is voted out. 
+
+## 7. Purpose of Administration
+
+1. Lock/close posts (and remove when content policy specifies).
+2. Ban users and deal with ban appeals (see the ban policy for more details).
+3. Manage the group name/cover photo and its promotion.
+
+Note that there are many tasks such as Hackbot administration, GitHub projects, etc. that do not need to be solely managed by admins.
+
+## 8. Deletion of Posts
+
+Posts should only be deleted in the following cases:
+
+1. Spam from hacked accounts or bots.
+2. Any posts violating Facebook's content policies will also be deleted.
+3. Content harassing a *specific* user. It is preferable to not remove this content (unless the harassment is extreme enough to violate Facebook's policies), so these types of posts should only be deleted on the request of the harassed user. Note that generally objectionable content that does not target a *specific* user will not be removed (unless it violates Facebook's content policies), and will only be closed instead.
+
+All other content that violates the content policy will be closed. The reason for this is to provide transparency as to what the admins deem to be bad content, and to leave examples of bad content in the group in an "archived" state so they can be used as examples of what not to post, as well as serving as precedent for closing future bad posts.
+
+## 9. Ban Policy
+
+1. You may be banned by violating any of the rules defined in the content policy. A warning may be given first depending on the which policy was violated and the severity of the violation.
+2. When a user is banned, they have a chance to return to the group. They must make a post in HX Meta to argue why they shouldn’t be banned, or to apologize and request that they be re-added. If the user is not apologetic or we (the admins) and the community (jointly) do not agree with the user’s justification for their behavior, they will be banned from HX Meta and no further appealing can take place. This will not require a formal voting process, and the admins will have the final say about the ban.
+3. Further policy violations in the main group after being re-added will result in immediate ban from both groups with no chance of appealing.
+4. The admins should keep track of the list of banned users, the date they were banned, and a link to or a copy of the content or behavior that led to their ban. How the admins choose to keep track of this data is up to the admins, but it must only be visible to current admins. Access to this data is transfered to new admins when they are elected and taken away from old admins when their term ends.
```

Comment:
"How the admins choose to keep track of this data is up to the admins, but it must only be visible to current admins."